Source/WebInspectorUI/UserInterface/Views/InlineSwatch.css

```diff
@@ -50,19 +50,19 @@
     background-repeat: no-repeat;
 }
 
-.inline-swatch:is(.cubic-bezier-timing-function, .spring-timing-function, .variable) {
+.inline-swatch:is(.cubic-bezier-timing-function, .spring-timing-function, .steps-timing-function, .variable) {
```

Comment:
I feel like timing-functions could deserve their own shared class

Reply:
i personally like the explicitness of this as it means i can do a code search for `cubic-bezier-timing-function` and find all the various places in CSS that i might need to make changes (which i actually did multiple times for this PR as i wasn't immediately aware of all the places where i might need to support `steps()` stuff)